_data/engine-cli/docker_stats.yaml

```diff
@@ -1,7 +1,96 @@
+
+      $ docker stats fervent_panini 5acfcb1b4fd1
+      CONTAINER           CPU %!M(MISSING)EM USAGE/LIMIT       MEM %!N(MISSING)ET I/O
+      5acfcb1b4fd1        0.00%! (MISSING)MiB/1.045 GiB   11.03%! (MISSING)kB/648 B
+      fervent_panini      0.02%! (MISSING)MiB/1.045 GiB   1.06%! (MISSING)B/648 B
+
+  Running `docker stats` on all running containers against a Windows daemon.
+
+      PS E:\> docker stats
+      CONTAINER           CPU %!P(MISSING)RIV WORKING SET    NET I/O             BLOCK I/O
+      09d3bb5b1604        6.61%! (MISSING)MiB           17.1 kB / 7.73 kB   10.7 MB / 3.57 MB
+      9db7aa4d986d        9.19%! (MISSING)MiB           15.2 kB / 7.65 kB   10.6 MB / 3.3 MB
+      3f214c61ad1d        0.00%! (MISSING)MiB           64 kB / 6.84 kB     4.42 MB / 6.93 MB
+
+  Running `docker stats` on multiple containers by name and id against a Windows daemon.
+
+      PS E:\> docker ps -a
+      CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
+      3f214c61ad1d        nanoserver          "cmd"               2 minutes ago       Up 2 minutes                            big_minsky
+      9db7aa4d986d        windowsservercore   "cmd"               2 minutes ago       Up 2 minutes                            mad_wilson
+      09d3bb5b1604        windowsservercore   "cmd"               2 minutes ago       Up 2 minutes                            affectionate_easley
+
+      PS E:\> docker stats 3f214c61ad1d mad_wilson
+      CONTAINER           CPU %!P(MISSING)RIV WORKING SET    NET I/O             BLOCK I/O
+      3f214c61ad1d        0.00%! (MISSING)MiB           76.3 kB / 7.92 kB   10.3 MB / 14.7 MB
+      mad_wilson          9.59%! (MISSING)MiB           27.6 kB / 8.81 kB   17 MB / 20.1 MB
+
+  ## Formatting
+
+  The formatting option (`--format`) pretty prints container output
+  using a Go template.
+
+  Valid placeholders for the Go template are listed below:
+
+  Placeholder  | Description
+  ------------ | --------------------------------------------
+  `.Container` | Container name or ID (user input)
+  `.Name`      | Container name
+  `.ID`        | Container ID
+  `.CPUPerc`   | CPU percentage
+  `.MemUsage`  | Memory usage
+  `.NetIO`     | Network IO
+  `.BlockIO`   | Block IO
+  `.MemPerc`   | Memory percentage (Not available on Windows)
+  `.PIDs`      | Number of PIDs (Not available on Windows)
+
+
+  When using the `--format` option, the `stats` command either
+  outputs the data exactly as the template declares or, when using the
+  `table` directive, includes column headers as well.
+
+  The following example uses a template without headers and outputs the
+  `Container` and `CPUPerc` entries separated by a colon for all images:
+
+  ```bash
+  $ docker stats --format "{{.Container}}: {{.CPUPerc}}"
+
+  09d3bb5b1604: 6.61%!
+(MISSING)  9db7aa4d986d: 9.19%!
+(MISSING)  3f214c61ad1d: 0.00%!
+(MISSING)  ```
```

Comment:
yup - working on fixing this ugly output - Not sure why the `(MISSING)` bits are being added 😞